waku/v0/message_test.go

```diff
@@ -40,3 +41,29 @@ func TestEncodeDecodeVersionedResponse(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, response.Response.Hash, v1resp.Hash)
 }
+
+func TestSendBundle(t *testing.T) {
+	rw1, rw2 := p2p.MsgPipe()
+	defer func() { handleError(t, rw1.Close()) }()
+	defer func() { handleError(t, rw2.Close()) }()
```

Comment:
These look nicer than the way I was implementing the same functionality. Inline is so fine 😎